packages/playwright-core/src/cli/driver.ts

```diff
@@ -90,13 +93,40 @@ function wireController(playwright: Playwright, wsEndpoint: string) {
       }
 
       if (message.method === 'inspect') {
-        for (const recorder of await allRecorders(playwright))
-          recorder.setMode(message.params.enabled ? 'inspecting' : 'none');
+        if (!message.params.enabled) {
+          for (const recorder of await allRecorders(playwright)) {
+            recorder.setHighlightedSelector('');
+            recorder.setMode('none');
+          }
+          return;
+        }
+
+        // Create browser if none.
+        const browsers = playwright.allBrowsers();
+        if (!browsers.length) {
+          process.send!({ method: 'error', params: { error: 'NEW BROWSER' } });
```

Comment:
Is this some kind of logging? Perhaps it's better to `console.log` and wire that on the client side?